java/pom.xml

```diff
@@ -36,8 +36,8 @@
        <artifactId>maven-compiler-plugin</artifactId>
        <version>2.5.1</version>
        <configuration>
-         <source>1.6</source>
-         <target>1.6</target>
+         <source>1.7</source>
+         <target>1.7</target>
```

Comment:
Is this just because 1.6 is out of support?

Reply:
Yes. I was unable to compile this Java version with my locally installed Java version 21. 1.7 is still outdated, but a little bit less worse imo.